kubetest2-gke/deployer/options/build.go

```diff
@@ -0,0 +1,70 @@
+package options
+
+import (
+	"fmt"
+	"regexp"
+
+	"sigs.k8s.io/kubetest2/pkg/build"
+)
+
+type BuildOptions struct {
+	Strategy      string `flag:"~strategy" desc:"Determines the build strategy to use either make or bazel."`
+	StageLocation string `flag:"~stage" desc:"Upload binaries to gs://bucket/ci/job-suffix if set"`
+	Version       string `flag:"~version" desc:"Use a specific GKE version e.g. 1.16.13.gke-400 or 'latest' or 'source' which will build kubernetes from source."`
```

Comment:
I feel like `--build` should imply `--version=source` ...?

Reply:
done.